build/17/maven/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
why has this file been added? the dockerfile still points to the start.sh file which is present in build/maven folder

Reply:
got it, have removed the file.